packages/headless/src/api/search/search/result.ts

```diff
@@ -27,15 +27,15 @@ export interface Result {
 
   /**
    * The hyperlinkable item URI.
-   * Notes: Use the clickUri value when you want to create hyperlinks to the item, rather than the uri or printableUri value.
+   * Notes: Use the clickUri value to create hyperlinks to the item, rather than the uri or printableUri value.
```

Comment:
Nitpick, but since you're modifying this string anyways:

```suggestion
   * Notes: Use the `clickUri` value to create hyperlinks to the item, rather than the `uri` or `printableUri` value.
```